snuba/clickhouse/optimize/optimize.py

```diff
@@ -381,8 +381,8 @@ def optimize_partitions(
     tracker: Optional[OptimizedPartitionTracker] = None,
     clickhouse_host: Optional[str] = None,
 ) -> None:
-    query_template = """\
-        OPTIMIZE TABLE %(database)s.%(table)s
+    query_template = f"""\
+        OPTIMIZE TABLE {database}.{table}
         PARTITION %(partition)s FINAL
```

Comment:
Any reason to remove `s` i.e. plural from databases and tables? Not sure if this is a big deal but want to call it out. For partitions, you seem to have kept the plural intact.